boggle-lib/src/main/java/com/boggle/FullWordCalculator.java

```diff
@@ -26,43 +23,22 @@
 
     public static void main(String[] args) {
         List<String> words = readFileInList("/Users/natashasinghvi/Documents/boggle/boggle-lib/src/main/java/com/boggle/popularWords.txt");
-        Boolean answer = checkIsWordFull("dog");
+        Boolean answer = checkIsWordFull("djlk");
```

Comment:
Delete main function in FullWordCalculator, if it's a test move it to src/test. 
Create a private variable Set<String> called dictionary. 
Initialize this variable in the class constructor by reading the file in. 
Expose a non static method that checks if a given word is contained in the dictionary.